docs/tutorials/lets_write_a_test.md

```diff
@@ -26,6 +26,26 @@ What needs to happen in this test:
 
 This command will generate a host file for our system under test (SUT). It will use vmpooler as hypervisor for the host. Please check out [this](https://github.com/puppetlabs/beaker/tree/master/docs/how_to/hypervisors) doc to learn more about hypervisors for beaker.
 
+## Creating the host configuration on at runtime
+
+When you don't want to store a static file, you can also let [beaker-hostgenerator](https://github.com/puppetlabs/beaker-hostgenerator) generate it on the fly. The filename is used as a host specification which comes down to the environment variable `BEAKER_setfile`. A simple example is:
+
+```
+BEAKER_setfile=centos7-64
```

Comment:
@ekohl `BEAKER_setfile` is not something that's parsed or used by beaker itself, it's a [beaker-rspec](https://github.com/puppetlabs/beaker-rspec/search?q=BEAKER_setfile&unscoped_q=BEAKER_setfile) property.

Since that's the case this doc should be moved there.

Reply:
It is how I influence the call to `beaker-hostgenerator` now, but you're right that `beaker-rspec` passes it in via the `hosts_file` option.